src/local/butler/scripts/setup.py

```diff
@@ -84,6 +84,18 @@
 THREAD_DELAY = 30.0
 """
 
+CENTIPEDE_TEMPLATE = """MAX_FUZZ_THREADS = 1
+MAX_TESTCASES = 2
+FUZZ_TEST_TIMEOUT = 8400
+TEST_TIMEOUT = 65
+WARMUP_TIMEOUT = 65
+BAD_BUILD_CHECK = False
+THREAD_ALIVE_CHECK_INTERVAL = 1
+REPORT_OOMS_AND_HANGS = True
+ENABLE_GESTURES = False
+THREAD_DELAY = 30.0
```

Comment:
@oliverchang do we know if this stuff is necessary? Seems weird (but understandable given CF's history) that we don't default to no for this stuff that is chrome only.

Reply:
Thanks for mentioning this!
Happy to remove this part if it is unnecessary, otherwise all addressed : )

Reply:
I think a lot of these are indeed necessary. Many of the defaults are still from early Chrome days.